### PR TITLE
LPS-133100 Back button not working

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java [hunk 1]
```diff
@@ -103,8 +103,19 @@ public DDMFormViewFormInstanceRecordsDisplayContext(
...
+
+		String redirect = ParamUtil.getString(_renderRequest, "redirect");
+
+		if (redirect.isEmpty()) {
+			DDMFormAdminDisplayContext ddmFormAdminDisplayContext =
+				(DDMFormAdminDisplayContext)_renderRequest.getAttribute(
+					WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+			redirect = String.valueOf(
+				ddmFormAdminDisplayContext.getPortletURL());
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java:115]: Hey @linolaoj and @eloisa-amorim,

Apparently, the `PortletURL` class already has a method (`toString`) to convert a Portlet URL to a String and, from what I could see, this method goes one step ahead when compared to the method `String.valueOf`. Besides, it also looks to be a common pattern on portal's code. In light of this, can any of you please confirm this information and if it's the case please use the method `toString` instead?

Thanks.

Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java:115]: Hi @natocesarrego , we were using the `toString`, but the `sourceFormat` task suggested using `String.valueOf`. That is the only reason why we are using `String.valueOf`. We can change back and try to run the SF again.

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java:115]: Not necessary, @linolaoj. :slightly_smiling_face: Let's follow SF suggestion. Thanks for your clarification. :wink:

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**